roles/pulp_health_check/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+pulp_admin_username: admin
```

Comment:
Shouldn't this be in defaults?

And isn't it not configurable in the rest of the installer or underlying Pulp? If so, it should be hardcoded. I'd rather not give users the impression they can change something they cannot.

Reply:
you are right! I'll hardcode it